### PR TITLE
small fixes,

### PR DESCRIPTION
**Pull request by alejandroEsc**
nothing complicated, removes  `when` warning message and small visual change to the max_k8s_version.sh

### CODE REVIEW

File: bin/max_k8s_version.sh [hunk 1]
```diff
@@ -23,7 +23,4 @@ source "${my_dir}/../lib/common.sh"
 trap control_c SIGINT
 LOCAL_KEV="kraken_action=max_k8s_version version_outfile=${OUTFILE}"
 
-ansible-playbook ${K2_VERBOSE} -i ansible/inventory/localhost \
-	ansible/max_k8s_version.yaml --extra-vars \
-	"${KRAKEN_EXTRA_VARS}${LOCAL_KEV}" \
-    || echo "max_version failed"
+ansible-playbook ${K2_VERBOSE} -i ansible/inventory/localhost ansible/max_k8s_version.yaml --extra-vars  "${KRAKEN_EXTRA_VARS} ${LOCAL_KEV}" || echo "max_version failed"
```
Comment (davidewatson) [inline bin/max_k8s_version.sh]: This reads better when it fits on my small screen. Also, please give your commits a useful description. These are records in the changelog.

Comment (davidewatson) [reply bin/max_k8s_version.sh]: Nack, my comment about the description. I didn't read after the first line. ;) Dorry!

Comment (alejandroEsc) [reply bin/max_k8s_version.sh]: LOL, you will find i try to details as much as possible in the commit, i think its severely important :)

**Comment by davidewatson** [comment]
https://github.com/samsung-cnct/k2/pull/800 does the same thing everywhere.

**Pull request closed without merging**